kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -43,7 +43,12 @@ grCollectionsPanel.controller('GrNodeCtrl',
     const ctrl = this;
     ctrl.saving = false;
     ctrl.deletable = false;
-    ctrl.addChild = childName => collections.addChildTo(ctrl.node, childName);
+    ctrl.formError = null;
+    ctrl.addChild = childName =>
```

Comment:
Changed, the reason I find it odd is that when we use return, the indentation is always harder to work out.